src/legacy/core_plugins/expressions/public/np_ready/public/mocks.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { ExpressionsSetup, ExpressionsStart, plugin as pluginInitializer } from '.';
+/* eslint-disable */
+import { coreMock } from '../../../../../../core/public/mocks';
+import { inspectorPluginMock } from '../../../../../../plugins/inspector/public/mocks';
+/* eslint-enable */
+
+const createExpressionsSetupMock = (): ExpressionsSetup => {
+  return {
+    registerFunction: jest.fn(),
+    registerRenderer: jest.fn(),
+    registerType: jest.fn(),
+    __LEGACY: {
+      functions: {
```

Comment:
use the new registries that np expressions service provides .... if we need access to more than register function please expose it.